tree-sitter-darklang/grammar.js

```diff
@@ -924,6 +924,7 @@ module.exports = grammar({
     //
     // Tuple
     tuple_type_reference: $ =>
+      // CLEANUP: make the parens optional
```

Comment:
We can leave this CLEANUP in but I don't hate the idea of requiring `Json.parse<(Int * Bool)>`, not allowing `Json.parse<Int * Bool>` - the former feels clearer. What do you think?

Reply:
actually, "tolerable parser" so nvm - the CLEANUP is good and we should allow the latter too. But maybe I'd prefer to _prettify_ the TypeRef with the parens :)